src/cdk/drag-drop/drag.ts

```diff
@@ -549,7 +553,7 @@ export class CdkDrag<T = any> implements AfterViewInit, OnDestroy {
     if (this._nextSibling) {
       this._nextSibling.parentNode!.insertBefore(this._rootElement, this._nextSibling);
     } else {
-      this._initialContainer.element.nativeElement.appendChild(this._rootElement);
+      this._initialRootElementParent.appendChild(this._rootElement);
```

Comment:
Yes, this is what the fix is for.

You said that this scenario is possible (i.e. tabs dragging) but you don't want to commit on an API.

So this will fix it for a single container, connected containers will have a different issue but with similar outcome.

Reply:
See how it helps with tabs: https://stackblitz.com/edit/angular-mgrsxx-tfeye2?file=app/tab-group-basic-example.ts